docs/notebooks/diagnostics/line_integrated_diagnostic.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>Just starting out by reading the notebook for now, but - I suppose the use of CartesianGrid here is more along the lines of "it's set up to work with CartesianGrid right now", rather than "It will never take a non-uniform grid"? Perhaps we could reword it to:</p><p> </p><p>"by providing a Grid object, representing the plasma to be measured (currently only CartesianGrid is supported)"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/1123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Actually, this should already work for non-uniform grids! CartesianGrids are just so much easier to set up. But maybe I should add an example with a non-uniform grid too...

Reply:
Ah, in that case, let's just reword it! :)